### PR TITLE
[BSv5] blocks/section row-col hierarchy fix

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to #1466
- Contributes to #906
- Fixes the width/max-width of `blocks/section`s
- Section types are now switched! The `type` argument can be either `container` (the **default**) or `row`.
  This is a **BREAKING CHANGE**. I've updated the docs and the CHANGELOG.
- Whitespace cleanup
- Fixes a long-standing bug of a blank paragraph at the end of the About page, since before BSv5 changes

**Preview**: https://deploy-preview-1472--docsydocs.netlify.app/about

---

#### Before

Notice the following problems (all of which are resolved by this PR):

- White space between the end of the last block and the footer
- Misformatting of bulleted lists
- Excessive width

> image

#### After

> image

### CODE REVIEW

File: layouts/shortcodes/blocks/section.html [hunk 1]
```diff
@@ -1,16 +1,14 @@
-{{ $_hugo_config := `{ "version": 1 }` -}}
```
Comment (deining) [inline layouts/shortcodes/blocks/section.html:1]: by removing this line, this PR also contributes to #939. Can you add this in the issue description, please?

**Review by deining — APPROVED** [review]
LGTM.

File: layouts/shortcodes/blocks/section.html (hunk 1 above)
Comment (chalin) [reply layouts/shortcodes/blocks/section.html:1]: Ah yes! Thanks for the reminder, ... and the review! 😄

**Comment by LisaFC** [comment]
Looks great, thanks Patrice!

File: userguide/content/en/about/index.md [hunk 2]
```diff
@@ -2,7 +2,6 @@
 title: About Docsy
 linkTitle: About
 menu: {main: {weight: 10}}
-layout: docs
```
Comment (chalin) [inline userguide/content/en/about/index.md:5]: @LisaFC et al.: prior to deleting line 5 above, there was a `_print` version of the about page, which pretty much looked like the non-print version. With line 5 gone, we've lot the `_print` version of the About page. Is that ok with you? Given that the two pages are identical AFAICT.

Comment (chalin) [reply userguide/content/en/about/index.md:5]: For the record, this is what the `_print` page looked like (excerpt), _before_ this PR was merged:

> image

Comment (LisaFC) [reply userguide/content/en/about/index.md:5]: I don't think that's a problem, given that the page was exactly the same.

**Pull request merged**